docs/training/example-2.adoc

```diff
@@ -0,0 +1,127 @@
+== Example 2: Running Shell Commands
+:includedir: _includes
+
+For this example we are going to explore kicking off Shell Commands from Beer Garden plugins. To accomplish
+this we have to select a Shell Command that best fits your environment. For this example we will use nslookup
+because it available on most operating systems (Mac OS, Ubuntu, Windows, Centos).
+
+[source, bash]
+.Example
+----
+nslookup python.org
+----
+
+Go ahead and run this in the command line somewhere to ensure that it works properly in your environment.
+You should get something like the following
+
+[source, bash]
+.Example
+----
+Server:     <DNS Server>
+Address:    <DNS Server Address>
+
+Non-authoritative answer:
+Name:    python.org
+Address: 45.55.99.72
+
+----
+
+Like the first example, lets build this out into a basic Python class.  For this example, we will provide
+the raw output but feel free to get creative with parsing the output.
+
+[source,python]
+.nslookup_plugin.py
+----
+
+import subprocess
+
+class NSLookupClient:
+
+    def execute_nslookup(self, domain):
+        process = subprocess.Popen(['nslookup', domain],
+                     stdout=subprocess.PIPE)
+
+        print(process.stdout.readlines())
+
+def main():
+	client = NSLookupClient()
+	client.execute_nslookup("python.org")
+
+if __name__ == "__main__":
+	main()
+----
+
+Instead of converting the function directly into a Beer Garden System Command, let's build a new command
+that will pass in the domain being queried. That way we can expose this new function in the UI.
+
```

Comment:
I think this is not the best approach - one of the "best" things about brewtils is the minimal amount of changes that you need to make to convert an existing class to a brewtils plugin. Adding a new function that calls an existing one is more verbose and would double the amount of methods if applied to a "real" class. Also, having a method who's only statement is to call another method is not great practice.

I think adding the decorators to the existing methods is a better way.